client-java/controller/src/main/java/org/evomaster/client/java/controller/internal/EMController.java

```diff
@@ -180,7 +180,18 @@ public Response getSutInfo(@Context HttpServletRequest httpServletRequest) {
             dto.infoForAuthentication = noKillSwitch(() -> sutController.getInfoForAuthentication());
             dto.sqlSchemaDto = noKillSwitch(() -> sutController.getSqlDatabaseSchema());
             dto.defaultOutputFormat = noKillSwitch(() -> sutController.getPreferredOutputFormat());
+
+            Set<ExternalServiceInfoDto> esDto = new HashSet<>();
+            sutController.getAdditionalInfoList().stream().map(e -> e.getExternalServices().stream()
```

Comment:
might add `List<AdditionalInfo> additionalInfos = noKillSwitch(() -> sutController.getAdditionalInfoList())` to ensure thread safe,
then process the esDto with the `additionalInfos`